docs/snippets/angular/page-story-with-args-composition.ts.mdx

```diff
@@ -0,0 +1,42 @@
+```ts
+// YourPage.stories.ts
+
+import { moduleMetadata } from '@storybook/angular'
+import { Story, Meta } from '@storybook/angular/types-6-0';
+
+import { CommonModule } from '@angular/common';
+
+import DocumentScreen from './YourPage.component';
+import DocumentList from './DocumentList.component';
+import DocumentHeader from './DocumentHeader.component';
+import PageLayout from './PageLayout.component';
+
+
+import * as PageLayoutStories from './PageLayout.stories';
+import * as DocumentHeaderStories from './DocumentHeader.stories';
+import * as DocumentListStories from './DocumentList.stories';
+
+export default {
+  component: DocumentScreen,
+  title: 'DocumentScreen',
+  decorators: [
+    moduleMetadata({
+      // imports components to allow component composition with Storybook
+      declarations: [DocumentList, DocumentHeader,PageLayout],
+      imports: [CommonModule],
+    }),
+  ],
+} as Meta;
+
+const Template: Story<DocumentScreen> = (args: DocumentScreen) => ({
+  component: DocumentScreen,
```

Comment:
same 
> it is necessary to redefine the component here as it is already defined in export default


I didn't look at all the snippets but `component` in stroy is deprecated